config/initializers/content_security_policy.rb

```diff
@@ -26,24 +26,10 @@
 Rails.application.config.content_security_policy do |policy|
   public_s3_url = ENV['S3_PUBLIC_URL'].present? ? "https://#{ENV['S3_PUBLIC_URL']}.s3.amazonaws.com/" : nil
 
-  # Get Superset base URL for CSP if available
-  superset_base_url = begin
-    Superset.superset_base_url if defined?(Superset)
-  rescue StandardError
-    nil
-  end
-
   policy.default_src(:self)
   policy.object_src(:none) # Prevents potentially dangerous browser plugins
   policy.base_uri(:self) # Only allows base URLs from your own domain, prevents cross-origin base URL injection
-  policy.form_action( # Protects against form-action hijacking
```

Comment:
removed since it seems be cause violations, it's unclear why but tabling for now sow we can get this in place